google-cloud-storage/samples/storage_create_bucket_turbo_replication.rb

```diff
@@ -24,7 +24,7 @@ def create_bucket_turbo_replication bucket_name:
                                   location: "ASIA",
                                   rpo:      "ASYNC_TURBO"
 
-  puts "Created bucket #{bucket.name} in #{bucket.location} with turbo replication set to #{bucket.rpo}."
+  puts "Created bucket #{bucket.name} in #{bucket.location} with the recovery point objective (RPO) set to #{bucket.rpo}."
```

Comment:
Seems like Rubocop isn't happy with the line length. Can you split this into two lines?

Something like this should work:
```rb
puts "Created bucket #{bucket.name} in #{bucket.location} with "
         + "the recovery point objective (RPO) set to #{bucket.rpo}."
```

Reply:
Updating. Thanks, @SandeepTuniki !